cinch/roles/jenkins_master/defaults/main.yml

```diff
@@ -13,18 +13,13 @@
 update_centers:
   - id: default
     url: "https://updates.jenkins-ci.org/{{ jenkins_version }}/update-center.json"
-# If set to true, Jenkins will be run over HTTPS (Jenkins on port 8443 with a
-# firewall forward from port 443). If you set this value to true, you MUST
-# override jenkins_ssl_cert and jenkins_ssl_key in a production system or you
-# will not see any improvements in security
-https_enabled: false
```

Comment:
Why have you completely gotten rid of this variable? SSL should still be optional.